### PR TITLE
Update kubectl get pods, svc output

### PR DESCRIPTION
**Pull request by rvennam**

### CODE REVIEW

File: content/docs/setup/kubernetes/install/kubernetes/index.md [hunk 1]
```diff
@@ -126,17 +133,19 @@ $ kubectl apply -f install/kubernetes/istio-demo-auth.yaml
...
+    istio-egressgateway-65f8c57947-2gb98      1/1     Running   0          6m
+    istio-galley-7fc6658847-kjnfq             1/1     Running   0          6m
+    istio-ingressgateway-6bbbccf8f4-wwknm     1/1     Running   0          6m
+    istio-pilot-5d76cd68bb-vj59r              2/2     Running   0          6m
+    istio-policy-6c86dc575d-nnz8w             2/2     Running   0          6m
+    istio-sidecar-injector-745794f57f-wjzdf   1/1     Running   0          6m
+    istio-telemetry-5df96b9557-7jb4d          2/2     Running   0          6m
+    istio-tracing-7dbc876499-g7lvc            1/1     Running   0          6m
+    kiali-5d579bf6d8-trp2k                    1/1     Running   0          6m
+    prometheus-6cdfbb4746-7494d               1/1     Running   0          6m
```
Comment (venilnoronha) [inline content/docs/setup/kubernetes/install/kubernetes/index.md]: Do you see any post-install pods with `Completed` status?

Comment (linsun) [inline content/docs/setup/kubernetes/install/kubernetes/index.md]: I'm seeing a bunch of completed too... you don't have any of these?

```
istio-cleanup-secrets-release-1.1-latest-daily-mt85x         0/1     Completed   0          3d2h
istio-egressgateway-7cb4ffdb75-rpcgh                         1/1     Running     0          47h
istio-galley-9d5986bc8-6shzf                                 1/1     Running     0          3d2h
istio-grafana-post-install-release-1.1-latest-daily-n4xcz    0/1     Completed   0          47h
istio-ingressgateway-5b6d8f9ff8-w96mz                        1/1     Running     0          3d2h
istio-init-crd-10-xnftm                                      0/1     Completed   0          3d2h
istio-init-crd-11-5x54g                                      0/1     Completed   0          3d2h
istio-init-crd-certmanager-10-8kkc6                          0/1     Completed   0          3d2h
istio-init-crd-certmanager-11-h8vg8                          0/1     Completed   0          3d2h
istio-pilot-86f75c7cdd-mrk4j                                 2/2     Running     0          47h
istio-security-post-install-release-1.1-latest-daily-dc4qd   0/1     Completed   0          3d2h
```

File: content/docs/setup/kubernetes/install/kubernetes/index.md [hunk 2]
```diff
@@ -102,16 +102,23 @@ $ kubectl apply -f install/kubernetes/istio-demo-auth.yaml
...
+    jaeger-query             ClusterIP      172.21.189.44    <none>          16686/TCP                                                                                                                 2m
+    kiali                    ClusterIP      172.21.231.125   <none>          20001/TCP                                                                                                                 2m
+    prometheus               ClusterIP      172.21.53.212    <none>          9090/TCP                                                                                                                  2m
+    tracing                  ClusterIP      172.21.66.18     <none>          80/TCP                                                                                                                    2m
+    zipkin                   ClusterIP      172.21.205.179   <none>          9411/TCP                                                                                                                  2m
     {{< /text >}}
```
Comment (linsun) [inline content/docs/setup/kubernetes/install/kubernetes/index.md]: I'm also seeing service graph on the list.

File: content/docs/setup/kubernetes/install/kubernetes/index.md (hunk 1 above)
Comment (rvennam) [reply content/docs/setup/kubernetes/install/kubernetes/index.md]: Strange - I don't see any jobs in my `get pods` output. I can try again on a new install.

Comment (venilnoronha) [reply content/docs/setup/kubernetes/install/kubernetes/index.md]: Are you using the latest daily build?

File: content/docs/setup/kubernetes/install/kubernetes/index.md (hunk 2 above)
Comment (rvennam) [reply content/docs/setup/kubernetes/install/kubernetes/index.md]: fixed

File: content/docs/setup/kubernetes/install/kubernetes/index.md (hunk 1 above)
Comment (rvennam) [reply content/docs/setup/kubernetes/install/kubernetes/index.md]: Fixed using the latest build. Thanks

**Pull request merged**